ext/node/ops/sqlite/backup.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+use std::borrow::Cow;
+use std::ffi::c_int;
+use std::time;
+
+use deno_core::OpState;
+use deno_core::op2;
+use deno_permissions::OpenAccessKind;
+use rusqlite::Connection;
+use rusqlite::backup;
+use serde::Deserialize;
+use serde::Serialize;
+
+use super::DatabaseSync;
+use super::SqliteError;
+use crate::NodePermissions;
+
+const DEFAULT_BACKUP_RATE: c_int = 5;
+
+#[derive(Serialize, Deserialize)]
+struct BackupOptions {
+  source: Option<String>,
+  target: Option<String>,
+  rate: Option<c_int>,
+}
+
+#[derive(Serialize, Deserialize)]
+struct BackupResult {
+  #[serde(rename = "totalPages")]
+  total_pages: c_int,
+}
+
+#[op2(stack_trace)]
+#[serde]
+pub fn op_node_database_backup<P>(
+  state: &mut OpState,
+  #[cppgc] source_db: &DatabaseSync,
+  #[string] path: String,
```

Comment:
I think this can be changed to `&str` (which avoids a clone in some cases I think)